### ISSUE TITLE
Load translation for a word/sentence depending on the context?

### ISSUE THREAD

**Issue by NVK15** [rubenv/angular-gettext#336]
Hello,

I have a situation where i have two different translations for the same word.
Example:
"Patients": "患者数", //present in file1.json
"Patients": "患者", //present in file2.json

These two translations are present in two different json files(file1.json and file2.json). But when i have the application loaded, both the translation json files are loaded in order. The "Patients" word has different translations depending on the context in two different HTML pages. Depending on the order in which the translation json files load, the word "Patients" gets translated.

How can i load the correct translation for the word depending on the context? Is there any configuration which we can do?

Looking forward for reply!

Thanks in advance.

**Comment by rubenv** [comment]
Yes, contexts: https://angular-gettext.rocketeer.be/dev-guide/annotate/#contexts
